src/styles/main.scss

```diff
@@ -1,3 +1,21 @@
+@import "./utils/variables.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/productCard.scss";
+@import "./blocks/container.scss";
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+}
+
 body {
   margin: 0;
+  box-sizing: border-box;
+}
+
+.main {
+  &__container {
+    margin: auto;
+  }
```

Comment:
Try to use external files for your tags to maintain the correct order of your final css file :)